doc/source/whatsnew/v0.20.0.txt

```diff
@@ -398,7 +398,32 @@ To convert a ``SparseDataFrame`` back to sparse SciPy matrix in COO format, you
 
    sdf.to_coo()
 
-.. _whatsnew_0200.enhancements.intervalindex:
+.. _whatsnew_0200.enhancements.style_excel:
+
+Excel output for styled DataFrames
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Experimental support has been added to export ``DataFrame.style`` to Excel using the ``openpyxl`` engine. (:issue:`15530`)
+
+For example, after running the following, ``styled.xlsx`` renders as below:
+
+.. ipython:: python
```

Comment:
after the ``df.iloc[0, 2] = np.nan``, do a ``df``, to show the df (in the docs).

add a block after this one to remove the generated file.

```
.. ipython:: python
   :suppress:
   import os
   os.remove('styled.xlsx')
```